omero/sysadmins/server-upgrade.rst

```diff
@@ -320,13 +320,42 @@ up to date to ensure that security updates are applied:
       $ # first, activate virtualenv where omero-py is installed. Then upgrade:
       $ pip install --upgrade 'omero-py>=\ |version_py|'
 
+.. _server_certificates:
 
 Server certificate
 ^^^^^^^^^^^^^^^^^^
-Since OMERO 5.6.2,  the recommended way to ensure that all OMERO server installations have, at minimum, a self-signed certificate is to use the `OMERO server certificate management plugin <https://github.com/ome/omero-certificates>`_. The plugin will generate or update your self-signed certificates and configure the OMERO.server. For the configuration to take effect, the server needs to be restarted.
+
+The server should be configured with at least a self-signed certificate to allow
+clients to establish secure connections.
+
+Since OMERO 5.6.2, the recommended way to ensure that all OMERO server installations have
+at minimum, a self-signed certificate is to use the
+`OMERO server certificate management plugin <https://github.com/ome/omero-certificates>`_.
```

Comment:
I think we should add ``omero-certificates`` in brackets since we refer it with that name in the reset of the documentation

or simply use 
```
the `omero-certificates <https://github.com/ome/omero-certificates>`_ plugin
```